pkgs/applications/virtualization/kata-containers/default.nix

```diff
@@ -0,0 +1,193 @@
+{ stdenv, buildGoPackage, fetchFromGitHub, fetchurl, pkgconfig, libseccomp, go
+, qemu }:
+
+let
+  version = "1.11.2";
+
+  # prepare source to have a structure of GOPATH
+  setSourceRoot = ''
+    mkdir -p "go/src/$(dirname "$goPackagePath")"
+    cp -R source "go/src/$goPackagePath"
+    export sourceRoot="go/src/$goPackagePath"
+  '';
+
+  # default make flags, set GOPATH, GOCACHE and install prefix
+  makeFlags = [
+    "GOPATH=$(NIX_BUILD_TOP)/go"
+    "GOCACHE=$(TMPDIR)/go-cache"
+    "PREFIX=$(out)"
+  ];
+
+  # default meta
+  meta = with stdenv.lib; {
+    homepage = "https://katacontainers.io/";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ offline ];
+    platforms = platforms.linux;
+  };
+
+  agent = stdenv.mkDerivation {
+    pname = "kata-agent";
+    inherit version;
+
+    src = fetchFromGitHub {
+      owner = "kata-containers";
+      repo = "agent";
+      rev = version;
+      sha256 = "6Zr7qwBuidJmKdZL2VGmRlKthWletdc0fRS5kA54CHM=";
+    };
+
+    goPackagePath = "github.com/kata-containers/agent";
+
+    inherit setSourceRoot;
+
+    nativeBuildInputs = [ pkgconfig go ];
+    buildInputs = [ libseccomp ];
```

Comment:
`seccomp` looks like it's off by default.

https://github.com/kata-containers/agent/blob/e01f28988780f43b1d023dd46272f334d5ec8230/Makefile#L79-L83

Reply:
Good catch, `seccomp` is now enabled, thanks!